pkg/pidfile/pidfile.go

```diff
@@ -33,6 +36,7 @@ func New(path string) (*PIDFile, error) {
 	if err := checkPIDFileAlreadyExists(path); err != nil {
 		return nil, err
 	}
+	system.MkdirAll(filepath.Dir(path), os.FileMode(0644))
```

Comment:
@diogomonica are these permissions correct? (I notice the file below is using them already, but just checking) :smile: